packages/ember-glimmer/tests/utils/shared-conditional-tests.js

```diff
@@ -283,7 +284,8 @@ export const ObjectTestCases = {
     this.assertText('T1F2');
   },
 
-  ['@test it tests for `isTruthy` on Ember objects if available']() {
+  // Marking as @htmlbars since POJOs can no longer masquerade as Proxies in glimmer2. See proxy tests below.
+  ['@htmlbars it tests for `isTruthy` on Ember objects if available']() {
```

Comment:
If this is not supported behavior (and that is basically what we are saying), we should just remove this test.